debian/control

```diff
@@ -3,7 +3,7 @@ Section: graphics
 Priority: optional
 Maintainer: Canon Inc. <sup-debian@list.canon.co.jp>
 Build-Depends: debhelper (>= 4.0.0), libcupsys2-dev | libcups2-dev,
-               libxml2-dev, libtiff4-dev, automake, autoconf,
+               libxml2-dev, libtiff5-dev, automake, autoconf,
```

Comment:
I think like cups and evince and others, this should use the generic `libtiff-dev`, which is provided by `libtiff5-dev`. Then it's more future proof if there's ever a `libtiff6-dev`.